### PR TITLE
set default dns result to ipv4 (node17,18)

### PR DESCRIPTION
**Pull request by hypervillain**
Node dns resolution was refactored starting at node v17, using ipv6 by default. This prevented to `node-fetch` localhost on 2 OSX machines

### CODE REVIEW

**Review by sintinilorenzo — APPROVED** [review]

**Comment by MarcMcIntosh** [comment]
Are we still using node-fetch?

**Comment by MarcMcIntosh** [comment]
Seems the only place where node-fetch / global fetch is used is in `server//src/api/screenshots/puppeteer.ts:74:    await fetch(screenshotUrl);`

Would it make more sense to remove it and use axios?

**Comment by hypervillain** [comment]
it does @MarcMcIntosh !

File: packages/slice-machine/package.json [hunk 1]
```diff
@@ -71,7 +71,6 @@
     "mime": "^2.5.2",
     "module-alias": "^2.2.2",
     "neverthrow": "^4.2.1",
-    "node-fetch": "^2.6.7",
```
Comment (MarcMcIntosh) [inline packages/slice-machine/package.json:74]: yasss

File: packages/slice-machine/package.json [hunk 2]
```diff
@@ -111,7 +110,6 @@
     "@types/mime": "^2.0.3",
     "@types/mock-fs": "^4.13.1",
     "@types/module-alias": "^2.0.1",
-    "@types/node-fetch": "^2.5.12",
```
Comment (MarcMcIntosh) [inline packages/slice-machine/package.json:114]: yass

File: packages/slice-machine/server/src/index.ts [hunk 3]
```diff
@@ -11,16 +11,12 @@ import bodyParser from "body-parser";
...
-import fetch from "node-fetch";
 
 // fake comment used to trigger the CI.
 
 declare let global: {
-  fetch: typeof fetch;
   appRoot: string;
 };
-
-global.fetch = fetch; // TODO: remove this bit it's still used in screen-shots
```
Comment (MarcMcIntosh) [inline packages/slice-machine/server/src/index.ts:23]: yass

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**